src/types/drupal/field_type.d.ts

```diff
@@ -102,3 +104,19 @@ export interface BreadCrumbLink {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   options?: any[]
 }
+
+/**
+ * Types for services and health services
+ */
+
+export interface HealthService {
+  entity: {
+    field_body: FieldFormattedText | null
+    field_service_name_and_descripti: TaxonomyTermHealthCareServiceTaxonomy & {
+      // Overide the description as string type in  DrupalTaxonomyTerm
+      description: FieldFormattedText
```

Comment:
Can this override be done on the taxonomy term type itself? It seems to match with what I see at https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/taxonomy_term/health_care_service_taxonomy/a9f98f2a-313e-49dc-b900-5a0000989ba6?resourceVersion=id%3A1224

Reply:
👍